blueflood-core/src/test/java/com/rackspacecloud/blueflood/io/serializers/HistogramSerializationTest.java

```diff
@@ -13,12 +13,10 @@
  *    See the License for the specific language governing permissions and
```

Comment:
Ah, I didn't know that. Good to know. Since I am planning to get rid of the serialization file this test uses, I went ahead and clean it up.